src/capture.rs

```diff
@@ -954,13 +957,23 @@ impl Capture {
                                              endpoint.device_address,
                                              fields.index)
                             },
-                            match (fields.length, transfer_data.len()) {
+                            match (fields.length, size) {
                                 (0, 0) => "".to_string(),
                                 (len, size) if size == len as usize => format!(
                                     ", {} {} bytes", action, len),
                                 (len, size) => format!(
                                     ", {} {} of {} requested bytes",
                                     action, size, len)
+                            },
+                            match (request_type, std_req, descriptor_type) {
+                                (Standard, GetDescriptor, String)
+                                    if size >= 4 && fields.index != 0 => format!(
+                                        ": '{}'",
+                                        WString::from_utf16le(
+                                            transfer_data[2..size].to_vec()
+                                        ).unwrap().to_utf8()
```

Comment:
Yeah good call, I'll do that.

Reply:
This will now display `Getting string descriptor [...]: Invalid UTF-16 string` if the UTF-16 fails to decode.

I think any more detail than that isn't going to fit well in the summary. We can add capture of string descriptor requests to the decoder and include them in the descriptor hierarchy pane - I think that's probably a better place to put detail on malformed descriptors.